### ISSUE TITLE
styler hook: Parameters not parsed correctly on Windows

### ISSUE THREAD

**Issue by lorenzwalthert** [lorenzwalthert/precommit#195]
It seems as  `'--style_transformers=tidyverse_style(scope = \"none\")'` and similar cannot be parsed correctly on Windows, see #186 and a couple of variants in 6c69c1cc93e45fc889a2e222aad916e712610bb4 specifically.

**Comment by lorenzwalthert** [comment]
Seems to be a problem in base R/powershell. We have a reproducible example here: https://github.com/lorenzwalthert/test-docopt

**Comment by lorenzwalthert** [comment]
New idea: revert the API change and pass style guide arguments as command line arguments, like this:
```yaml
        args: [--style_pkg=styler, --style_fun=tidyverse_style, --scope=spaces, --indent_by=2]    
```
This way, the we revert to the API of styler <= 1.2, i.e. `style_pkg` and _style_fun` have exact same meaning as before, we just add new arguments. That way, we get rid of the (double)quoting issues in powershell and provide a more CLI-like way of specifying the style guide.

**Comment by lorenzwalthert** [comment]
Closed with #183. Follow up on #178.
